index.html

```diff
@@ -139,317 +139,339 @@
       </ul>
     </nav>
 
-    <div class="content-wrapper">
-      <section class="main-banner">
-        <section class="notif-banner">
-          <p class="notif">
-            <span>New</span>
-            <strong>1 year Anniversary</strong>
-            -- New features available for you
-            <a href="/releases.html"> Check them out! ➡️ </a>
+    < class="content-wrapper">
+    <section class="main-banner">
+      <section class="notif-banner">
+        <p class="notif">
+          <span>New</span>
+          <strong>1 year Anniversary</strong>
+          -- New features available for you
+          <a href="/releases.html"> Check them out! ➡️ </a>
+        </p>
+      </section>
+    </section>
+
+    <main>
+      <h1 class="welcome-title">Welcome to Real Dev Squad</h1>
+      <img
+        class="logo-img"
+        src="img/Real-Dev-Squad@1x.png"
+        alt="Real Dev Squad logo"
+      />
+      <section class="social-media-links">
+        <a
+          class="social-media-link"
+          href="https://www.linkedin.com/company/real-dev-squad/"
+          target="_blank"
+          rel="noreferrer"
+        >
+          <button class="social-media-links-btn">
+            <img
+              class="social-media-logo"
+              src="img/linkedin.svg"
+              alt="linkedinlogo"
+            /></button
+        ></a>
+        <span>|</span>
+        <a
+          class="social-media-link"
+          href="https://twitter.com/realdevsquad"
+          target="_blank"
+          rel="noreferrer"
+        >
+          <button class="social-media-links-btn">
+            <img
+              class="social-media-logo"
+              src="img/twitter.svg"
+              alt="twitterlogo"
+            />
+          </button>
+        </a>
+        <span>|</span>
+        <a
+          class="social-media-link"
+          href="https://www.facebook.com/Real-Dev-Squad-108713777585062"
+          target="_blank"
+          rel="noreferrer"
+        >
+          <button class="social-media-links-btn">
+            <img
+              class="social-media-logo"
+              src="img/facebook.svg"
+              alt="facebooklogo"
+            />
+          </button>
+        </a>
+        <span>|</span>
+        <a
+          class="social-media-link"
+          href="https://instagram.com/realdevsquad"
+          target="_blank"
+          rel="noreferrer"
+        >
+          <button class="social-media-links-btn">
+            <img
+              class="social-media-logo"
+              src="img/instagram.svg"
+              alt="instagramlogo"
+            />
+          </button>
+        </a>
+      </section>
+
+      <section class="description-what-is">
+        <h1 class="description-what-is-title">What is Real Dev Squad?</h1>
+        <div class="description-what-is-content">
```

Comment:
suggestion: could have used better names for classes

Reply:
Hey, @vinayak-trivedi I have  changed the names.